src/legacy/server/config/schema.js

```diff
@@ -144,6 +144,7 @@ export default () =>
 
     ops: Joi.object({
       interval: Joi.number().default(5000),
+      cGroupOverrides: HANDLED_IN_NEW_PLATFORM,
```

Comment:
We can probably remove old the schema up on lines 52-66 now?

Reply:
Right. I had to add the explicit definition in `ops` though, as 

https://github.com/elastic/kibana/blob/c528f24f8ba7fdb88c1f0c75898f2567ca053b0a/src/legacy/server/status/lib/metrics.js#L119-L120

is throwing error when trying to access the property if the schema is undefined (which make sense, `HANDLED_IN_NEW_PLATFORM` is only meant to be used for config values that are ONLY used in KP)